projects/mysql-server/targets/fuzz_docommand.cc

```diff
@@ -40,6 +40,7 @@ extern "C" int LLVMFuzzerInitialize(const int* argc, char*** argv) {
 }
 
 
+//TODO fix with 8.21
```

Comment:
// FIXME: Fix .... with 8.21
USE FIXME: or TODO(catenacyber): 
and please add what is broken in fix comment.

Reply:
Ok